doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -251,7 +251,7 @@ Non-resilient applications are sensitive to platform impairments on Compute like
 
 <p align="center"><b>Table 3-1:</b> Categories of applications, requirements for scheduling pods and Kubernetes features</p>
 
-Kubernetes clusters using above enhancements can implement worker nodes with "bare metal" servers (running Container Runtime in Linux host Operating System) or with virtual machines (VMs, on hypervisor).
+Kubernetes clusters using above enhancements can implement worker nodes with "bare metal" servers (running Container Runtime in Linux host Operating System directly on the hardware without any hypervisor) or with virtual machines (VMs, on hypervisor).
```

Comment:
Worker nodes can also utilise Windows OS. Linus has embedded KVM; ESXi and similar other are a combination OS + hypervisor -- although Type 1 hypervisors they are sometimes referred to as OSs. 
```suggestion
Kubernetes clusters using above enhancements can implement worker nodes with "bare metal" servers (running Container Runtime in the host Operating System directly on the hardware without need for any hypervisor capabilities) or with virtual machines (VMs, on hypervisor).
```